### DISCUSSION TITLE
Directories of files

### DISCUSSION THREAD

**Discussion by cfidler** [microsoft/presidio#975] [Q&A]
Hi - I'm trying to see if there is a way to run Presidio on directories of files - like 10,000 files. Has anyone been able to do that? My hope is to build something like the streamlit demo app but that the output would be a list of filenames containing the flagged texts.

**Comment by omri374** [comment]
Hi @cfidler, check out the Presidio CLI: https://github.com/microsoft/presidio/tree/main/presidio-cli

Cc @walkowif
